client/header/activity-panel/index.js

```diff
@@ -86,12 +86,14 @@ export const ActivityPanel = ( { isEmbedded, query, userPreferencesData } ) => {
 	};
 
 	const closePanel = () => {
-		setCurrentTab( '' );
 		setIsPanelOpen( false );
 	};
 
 	const clearPanel = () => {
-		setIsPanelSwitching( 'false' );
+		if ( ! isPanelOpen ) {
+			setIsPanelSwitching( false );
+			setCurrentTab( '' );
```

Comment:
Setting the `currentTab` as empty here works nicer, as it would continue to show the data as the panel slides closed. Once closed it will remove the rendered items.